app/models/record.rb

```diff
@@ -47,6 +47,22 @@ def self.attendance_by_month!(branch_id)
         data[key] = average_hours / value.length
       end
     end
-    data
+    data = data.to_a.reverse.to_h
+  end
+
+  def self.absence_by_month!(branch_id)
+    daysxd = 31
+    branch = Branch.find(branch_id)
+    employees = branch.employees
+    expected_attendances = employees.count * daysxd
```

Comment:
You use daysxd as a constant of value 31, but what do you think about that it would be better if daysxd calculates the value according to the current month?